application/backend/src/main/java/io/typerefinery/websight/models/components/widgets/Tickervariant.java

```diff
@@ -0,0 +1,93 @@
+package io.typerefinery.websight.models.components.widgets;
+
+import static org.apache.sling.models.annotations.DefaultInjectionStrategy.OPTIONAL;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import lombok.Getter;
+
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Default;
+import org.apache.sling.models.annotations.Exporter;
+import org.apache.sling.models.annotations.ExporterOption;
+import org.apache.sling.models.annotations.Model;
+
+import io.typerefinery.websight.models.components.BaseComponent;
+
+
+@Model(adaptables = Resource.class, resourceType = { "typerefinery/components/widgets/charts" }, defaultInjectionStrategy = OPTIONAL)
+@Exporter(name = "jackson", extensions = "json", options = { 
+    @ExporterOption(name = "MapperFeature.SORT_PROPERTIES_ALPHABETICALLY", value = "true"),
+    @ExporterOption(name = "SerializationFeature.WRITE_DATES_AS_TIMESTAMPS", value = "false") 
+})
+public class Tickervariant extends BaseComponent {

```

Comment:
Change this Ticker

Reply:
As there is work going on the ticker component regarding flow(totaljs).As of now let the name be unique,so there might not have conflicts